lib/admin.php

```diff
@@ -174,6 +175,13 @@ function scripts() {
 							array(), 
 							'0.1.0' 
 						);
+
+		$atts = array(
+			'nonce' => wp_create_nonce( 'genesis_love' ),
+			'ajaxurl' =>  admin_url('admin-ajax.php'),
```

Comment:
should have some spaces. `admin_url( 'admin-ajax.php' );`. Same for the next few lines.